modules/RTC/TPCUtils.js

```diff
@@ -122,8 +122,19 @@ export class TPCUtils {
                 if (mline.type === 'video' && i !== idx) {
                     sdp.media[i].rids = undefined;
                     sdp.media[i].simulcast = undefined;
+
+                    // eslint-disable-next-line camelcase
+                    sdp.media[i].simulcast_03 = undefined;
                 }
             });
+
+            // sdp-transform seems to be inserting a= simulcast line for all the video m-lines even if we make
```

Comment:
are we not in control of sdp-transform? is it not a better place for a fix? also what does `simulcast_03` stand for?

Reply:
nope we are not in control of sdp-transform. `simulcast_03` is the old simulcast draft 03 which is implemented in FF older versions.

Reply:
returning early doesn't modify the behavior, if we detect that the a=simulcast line isalready  present in the first video m-line, then we don't have to try to set it again.